### DISCUSSION TITLE
Is there a custom plugin to add a backup to s3 via Kong?

### DISCUSSION THREAD

**Discussion by rohit-gulati-fourkites** [Kong/kong#13850] [Ideas and feature requests]
I want to upload the API Payloads to S3

**Comment by xianghai2** [comment]
Hi @rohit-gulati-fourkites , you may take a look into aws-lambda plugin.
https://docs.konghq.com/hub/kong-inc/aws-lambda/
https://github.com/Kong/kong/tree/master/kong/plugins/aws-lambda

**Reply by rohit-gulati-fourkites** [reply]
Thanks! This is helpful.
